docs/aerodromes/classc/Williamtown.md

```diff
@@ -21,6 +21,62 @@ WLM ADC owns the airspace within the WLM MIL CTR A (`SFC`-`A050`). This may be a
   <figcaption>WLM ADC Airspace</figcaption>
 </figure>
 
+### SID Selection
+Aircraft planned via **KEXAR**, **LIMLO**, **MONDO**, **UBSON**, shall be assigned the **Procedural SID** that terminates at the appropriate waypoint.  
+Aircraft who are not planned via those points or who are negative RNAV may be assigned a RADAR SID or a visual departure. 
+
+#### Coded Departures (Fast Jets)
+Visual Departures are commonly requested by high performance military jets and often processed in the form of a coded departure (e.g. CLASSIC1 departure), which can be found in the [FIHA AD2 Supps](https://ais-af.airforce.gov.au/australian-aip){target=new} for YWLM. These coded departures provide a corridor with altitude constraints to join the Military Training Areas east and west of YWLM, avoiding civilian traffic inbound to YSSY. 
+
+| Tracking Gate | Departure | Initial Constraint | Route |
+| ----- | ---------| --------------- | ------------------------ |
+| G1-G2 | PEPPER 1 | `A080` at ERVEV | ERVEV G1/G2 (Transition) 
+| G3-G7 | CLASSIC 1| `F125` at G3-G7 |
+| G9    | RADAR    | `F125` at G9    |
+| UKDIP | CLASSIC 1| `A090` at UKDIP |
+| OLVEP | CLASSIC 1| `F130` at OLVEP |
+
+!!! note
+    ACFT Departing RWY 12 for G3-7 are required to avoid SAWR on the Classic 1.  
+    ACFT Departing on Classic 1 will be issued a direction of turn during take off clearance, preferably to the south in order not to impede on the SAWR boundaries on the departure.
+
+!!! phraseology
+    WGTL21 plans to enter M550 via Gate 4 for Military Training.  
+    **WLM ADC**: "WGTL21, cleared to YWLM via Gate 4, flight planned route. Runway 12, Classic 1 Departure. Climb to F125. Squawk 5001, Departure Frequency 133.300"  
```

Comment:
In line with the other discussion point, I'm assuming that WGTL21 wouldn't file Gate 4 in their flight plan, so should we change the example to instead be: `cleared to M550 via Gate 4` (no flight planned route)?

Reply:
> In line with the other discussion point, I'm assuming that WGTL21 wouldn't file Gate 4 in their flight plan, so should we change the example to instead be: `cleared to M550 via Gate 4` (no flight planned route)?

They can technically plan via a Gate, for example for Gate 4, they could plan via WLM088025 on their flight plan. This could be how they do it IRL, i'm not sure

Reply:
Yeah I guess in that case it's a bit more straightforward